config/semantics/java.yaml

```diff
@@ -27,6 +27,9 @@ semantics:
   - signature: "com.braintreepayments.api.models.PaymentMethodNonce.getNonce:<unresolvedSignature>(0)"
     flow: "0->-1"
 
+  - signature: "com.braintreepayments.api.models.PaymentMethodNonce.getNonce:java.lang.String(0)"
```

Comment:
@gaurav-gogia As the count of argument is 0, we can remove 0 from the fully qualified signature